### DISCUSSION TITLE
Import  template not working

### DISCUSSION THREAD

**Discussion by alelordelo** [GrapesJS/grapesjs#4614] [Q&A]
Hello community,

First, apologies if the question is dumb! I come form Xcode/iOS world, so I am new to web. : )

I am testing the editor, and copy from Codepen:
https://codepen.io/julesforrest/pen/oeVKjg

<img width="1440" alt="Screenshot 2022-09-24 at 19 43 53" src="https://user-images.githubusercontent.com/17708751/192111606-eee44a7c-bbd8-47db-9150-dd1552ed9a31.png">

And then pasted into GrapeJS. As it doesn't have a separate slot for HTML and CSS, I copied the HTML as it is, and then pressed enter and copied the CSS.

The problem is that the CSS gets into the HTML column.

<img width="1440" alt="Screenshot 2022-09-24 at 19 49 36" src="https://user-images.githubusercontent.com/17708751/192111823-edddd6b0-7f65-47b9-b735-20a3f89da816.png">

Am I doing something stupid here? 

Cheers!

**Comment by artf** [comment]
You have to paste CSS inside the `<style>` tag.

**Reply by alelordelo** [reply]
thanks @artf !

Works better now, but still not  correct CSS style applied. 

Here is the codepen:
https://codepen.io/julesforrest/pen/oeVKjg

<img width="2194" alt="Screenshot 2022-09-26 at 11 22 40" src="https://user-images.githubusercontent.com/17708751/192242642-2351aacc-fcf0-4582-a863-32bea6692552.png">

And bellow the results in GrapeJS:
Any idea what could be wrong?

<img width="2185" alt="Screenshot 2022-09-26 at 11 22 27" src="https://user-images.githubusercontent.com/17708751/192242483-1f72a6d9-1f71-451f-a402-9860c4ee0edc.png"> the



<img width="2191" alt="Screenshot 2022-09-26 at 11 21 53" src="https://user-images.githubusercontent.com/17708751/192241642-9aedb0e6-179d-4fbc-b9b5-65d77fa468a0.png">
